src/main/kotlin/com/amazon/opendistroforelasticsearch/indexmanagement/indexstatemanagement/transport/action/explain/ExplainResponse.kt

```diff
@@ -31,29 +31,34 @@ open class ExplainResponse : ActionResponse, ToXContentObject {
     val indexNames: List<String>
     val indexPolicyIDs: List<String?>
     val indexMetadatas: List<ManagedIndexMetaData?>
+    // val totalManagedIndices: Int
```

Comment:
Is commenting this out intentional or for testing?

Reply:
cleaned